### ISSUE TITLE
Confusion on the Labelling of High vs. Low SE

### ISSUE THREAD

**Issue by patmendoza6745** [MaHuanAAA/SemanticEnergy#1]
Hi there,

In the notebook, `semantic_energy.ipynb`. This line:
```
plt.xticks([min(normalized_scores_logits), max(normalized_scores_logits)], ['High', 'Low'])
```
confuses me. The paper states that lower energy => more reliability, yet we see a graph where incorrect answers skew "High" SE but doesn't that really mean it's normalized logit score is low according to the code above? Are we saying that incorrect answers have lower SE? Please let me know if I'm misinterpreting this or missing something.

EDIT: I see now that higher logits means lower SE.
